src/PlusVolumeReconstruction/vtkPlusVolumeReconstructor.cxx

```diff
@@ -654,46 +651,32 @@ PlusStatus vtkPlusVolumeReconstructor::SaveReconstructedVolumeToMetafile(const s
       return PLUS_FAIL;
     }
   }
-  return SaveReconstructedVolumeToMetafile(volumeToSave, filename, useCompression);
+  return SaveReconstructedVolumeToFile(volumeToSave, filename, useCompression);
 }
 
 //----------------------------------------------------------------------------
-PlusStatus vtkPlusVolumeReconstructor::SaveReconstructedVolumeToMetafile(vtkImageData* volumeToSave, const std::string& filename, bool useCompression/*=true*/)
+PlusStatus vtkPlusVolumeReconstructor::SaveReconstructedVolumeToFile(vtkImageData* volumeToSave, const std::string& filename, bool useCompression/*=true*/)

```

Comment:
We cannot just delete a method. We can deprecate it now and remove in 1-2 years.